onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.h

```diff
@@ -115,10 +103,22 @@ class ModelBuilder {
   // the given node must be in the underlying graph_viewer
   const NodeUnit& GetNodeUnit(const Node* node) const;
 
+  int32_t GetEffectiveFeatureLevel() const { return nnapi_effective_feature_level_; }
+
+  // Just for Debugging
```

Comment:
I have some reservations with this approach. it's adding a public non-const member function, and one that is expected to be used in quite a specific way.

might be ok if we put `#ifndef NDEBUG` around the whole function and its usages and document the expected usage clearly.

@skottmckay what do you think?